utils/variables.js

```diff
@@ -714,6 +714,16 @@ const IT_3101 = [
         code: "",
       },
     },
+  },
+    {
```

Comment:
remove extra {

Reply:
remove that in your local then type

git add .
git commit -m "update changes"
git push